addons/web/static/src/legacy/js/fields/relational_fields.js

```diff
@@ -2772,6 +2772,70 @@ var FieldMany2ManyTagsAvatar = FieldMany2ManyTags.extend({
     },
 });
 
+const M2MAvatarMixin = {
+    visibleAvatarCount: 3, // number of visible avatar
```

Comment:
Seems like your number contains the gray '+X'. It feels like a var called `visibleAvatarCount` should only contains the number of actually visible avatars.

Reply:
@SimonGenin Actually if there are 3 Avatars then we display all 3 avatar images, if there are more than 3 avatar then we display 2 avatar image and +X, so I think it is perfectly alright, don't you think so?

Reply:
@msh-odoo Hum, that's true in this case yes. Ok let's keep this